requirements.txt

```diff
@@ -1,7 +1,7 @@
 autopep8
 backports.unittest-mock
 click
-docker[ssh]>=3.6.0,<4.0
```

Comment:
>4.0 [](start = 20, length = 3)

do you know why was docker 4.0 an issue before?

Reply:
it was not. this version was a dependent on pyyaml 4.1. docker 4.4.4 and greater works with pyyaml 5.4